HackYourFuture-CPH/kp-toolbox-fp-class21
# Fp21 55 use more dynamic class naming logic for streak out categories

# Description

modified the route to get the categories of the tools with other data from tools table, and make streak out categories  name with dynamic class naming.

Fixes # (issue)

# How to test?
0-) run "docker compose up"
1-)run "yarn start server" in the backend.
2-) run "yarn start" in the frontend.
3-) you can see the categories in the tool cards "tool items".


# Checklist
